### PR TITLE
refine readme

### PR DESCRIPTION
**Pull request by baiyubin**
### 变更内容
- 添加英文readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,90 @@
...
-> - gopkg.in/check.v1 (gopkg.in/check.v1)
-
-## 快速使用
-#### 获取命令列表
+
+### [README of Chinese](https://github.com/aliyun/ossutil/blob/master/README-CN.md)
+
+## About
+- This tool is developed with Go and built on the official GO SDK 1.6.0 of OSS [Alibaba Cloud Object Storage Service](http://www.aliyun.com/product/oss/).
+- Alibaba Cloud Object Storage Service (OSS) is a cloud storage service provided by Alibaba Cloud, featuring a massive capacity, security, a low cost, and high reliability.
```
Comment (abo321) [inline README.md]: => featuring massive capacity, security, low cost and high reliability.

File: README.md [hunk 2]
```diff
@@ -1,87 +1,90 @@
...
+- The OSS can store any type of files and therefore applies to various websites, development enterprises and developers.
```
Comment (abo321) [inline README.md]: Following will be better.

OSS can store any type of files. It applies to various ...

File: README.md [hunk 3]
```diff
@@ -1,87 +1,90 @@
...
+- This tool aims to provide a convenient-to-use command line means for users to manage data in the OSS.
```
Comment (abo321) [inline README.md]: 去掉means, the

File: README.md [hunk 4]
```diff
@@ -1,87 +1,90 @@
...
-#### 获取命令列表
+
+### [README of Chinese](https://github.com/aliyun/ossutil/blob/master/README-CN.md)
+
+## About
+- This tool is developed with Go and built on the official GO SDK 1.6.0 of OSS [Alibaba Cloud Object Storage Service](http://www.aliyun.com/product/oss/).
+- Alibaba Cloud Object Storage Service (OSS) is a cloud storage service provided by Alibaba Cloud, featuring a massive capacity, security, a low cost, and high reliability.
+- The OSS can store any type of files and therefore applies to various websites, development enterprises and developers.
+- This tool aims to provide a convenient-to-use command line means for users to manage data in the OSS. 
+- The current version provides features to list and delete multipart upload tasks.
```
Comment (abo321) [inline README.md]: 去掉features

File: README.md [hunk 5]
```diff
@@ -1,87 +1,90 @@
...
+#### Others
+You can use `./ossutil help cmd` to view the help documentation for the command you want to use. 
 
-### 测试
-> - 进入go工程目录下的src目录，修改github.com/aliyun/ossutil/lib/command_test.go里的endpoint、AccessKeyId、AccessKeySecret、STSToken等配置。
-> - 请在lib目录下执行`go test`。
+## Notes
+### Run OSSUTIL
+- First, configure your Go project directory. 
+- Use the `go get` command to get the library that the project is dependent on.
```
Comment (abo321) [inline README.md]: => Use `go get`

File: README.md [hunk 6]
```diff
@@ -1,87 +1,90 @@
...
 
-### 测试
-> - 进入go工程目录下的src目录，修改github.com/aliyun/ossutil/lib/command_test.go里的endpoint、AccessKeyId、AccessKeySecret、STSToken等配置。
-> - 请在lib目录下执行`go test`。
+## Notes
+### Run OSSUTIL
+- First, configure your Go project directory. 
+- Use the `go get` command to get the library that the project is dependent on. 
+- Run `go get github.com/aliyun/ossutil`. 
+- Enter the *src* directory under the Go project directory and build code to generate the OSSUTIL tool. For example, on Linux, you can run `go build github.com/aliyun/ossutil/ossutil.go`.
```
Comment (abo321) [inline README.md]: build code => build

File: README.md [hunk 7]
```diff
@@ -1,87 +1,90 @@
...
+- Use the `go get` command to get the library that the project is dependent on. 
+- Run `go get github.com/aliyun/ossutil`. 
+- Enter the *src* directory under the Go project directory and build code to generate the OSSUTIL tool. For example, on Linux, you can run `go build github.com/aliyun/ossutil/ossutil.go`. 
+- Refer to the example above to run the OSSUTIL tool. 
 
-### 源码
-> - https://github.com/aliyun/ossutil
+### Test
+- Enter the *src* directory under the Go project directory and modify the endpoint, AccessKeyId, AccessKeySecret and STSToken configuration items in the *github.com/aliyun/ossutil/lib/command_test.go*. 
+- Run the `go test` command under the *lib* directory.
```
Comment (abo321) [inline README.md]: => Run `go test` under *lib* directory.

Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 6 above)
Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (baiyubin) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (baiyubin) [reply README.md]: fixed

**Pull request merged**